aiogram/dispatcher/event/handler.py

```diff
@@ -49,10 +60,10 @@ class HandlerObject(CallableMixin):
     callback: HandlerType
     filters: Optional[List[FilterObject]] = None
 
-    def __post_init__(self):
+    def __post_init__(self) -> None:
         super(HandlerObject, self).__post_init__()
-
-        if inspect.isclass(self.callback) and issubclass(self.callback, BaseHandler):
+        # TODO: by types callback must be Callable or BaseHandler, not Type[BaseHandler]
+        if inspect.isclass(self.callback) and issubclass(self.callback, BaseHandler):  # type: ignore
```

Comment:
issubclass expect class-object, but self.callback can be BaseHandler instance, not class

Is it error in type?

Reply:
Is not an error in type because we are registering subclasses of BaseHandler instead of instances of that classes

Example:
```python3
@router.message_handler()
class MyMessageHandler(MessageHandler):
    async def handle(self):
        ...

```